.github/workflows/release-pr.yml

```
@@ -0,0 +1,69 @@
+name: Create Release PR
+on:
+  create:
+    branches:
+    - 'release/v**'
+
+jobs:
+  build:
+    name: Create Release PR
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Calculate Tag
+      run: |
+        t=$(echo ${{ github.ref }} | sed -ne 's/.*\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p')
+        if [ -z "$t" ]; then
+        echo Malformed release branch name; failed to extract semVer tag
+        return 1
```

Should echo the branch name for diagnosing.  Tabbing seems off.  Oh, and you should quote the string as this will attempt to echo and then execute `failed` as a command.
```suggestion
          echo "Malformed release branch name '${{ github.ref }}': failed to extract semVer tag"
          return 1
```

(I hope the `${{ x }}` works within strings?)

---

done